extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js

```diff
@@ -1498,4 +1499,24 @@ describes.realWin('additional amp-ad-network-doubleclick-impl',
           });
         });
       });
+
+      describe('#preconnect', () => {
+        beforeEach(() => {
+          element = createElementWithAttributes(doc, 'amp-ad', {
+            'width': '200',
+            'height': '50',
+            'type': 'doubleclick',
+          });
+          doc.body.appendChild(element);
+          impl = new AmpAdNetworkDoubleclickImpl(element);
+        });
+
+        it('should preload nameframe', () => {
```

Comment:
Isn't this testing that we're preloading safeframe, not nameframe?

(Preloading safeframe is the intended behavior here, though, since this is doubleclick)

Reply:
Done